allennlp/nn/beam_search.py

```diff
@@ -103,16 +103,15 @@ def search(
             A function that is responsible for computing the next most likely tokens,
             given the current state and the predictions from the last time step.
             The function should accept two arguments. The first being a tensor
-            of shape `(group_size,)`, representing the index of the predicted
```

Comment:
Why change `group_size` to `batch_size`?

Reply:
I think it's same in conception but different in name. Unfortunately I am wrong.

Reply:
fix